src/main/java/core/basesyntax/Hello_World.java

```diff
@@ -1,46 +0,0 @@
-package core.basesyntax;
-
-import core.basesyntax.Model.Dog;
-import core.basesyntax.Model.Cat;
-
-import java.time.LocalDate;
-import java.util.Objects;
-
-public class Hello_World
```

Comment:
you should refactor this class so that it passes style check
you have removed it - please move it back and refactor